### PR TITLE
Popup when being pulled shows who is pulling you

### PR DESCRIPTION
**Pull request by Plykiya**
## About the PR

Adds a popup that indicates the identity of who is pulling you

## Why / Balance

Too often people will attempt to drag people out of the evac shuttle or into emagged recyclers because there is no way OOC to know who is actually dragging you, despite it being information you would 100% be aware of IC

## Technical details

Shows a popup to the entity being pulled with the visible identity name in the popup message

## Media

https://github.com/user-attachments/assets/5393021a-46de-45d2-a183-a28543ffaa4f

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: You are now notified of who is pulling you when being pulled.

### CODE REVIEW

File: Content.Shared/Movement/Pulling/Systems/PullingSystem.cs [hunk 1]
```diff
@@ -514,6 +515,10 @@ public bool TryStartPull(EntityUid pullerUid, EntityUid pullableUid,
         Dirty(pullerUid, pullerComp);
         Dirty(pullableUid, pullableComp);
 
+        var pullingMessage =
+            Loc.GetString("getting-pulled-popup", ("pulling", Identity.Entity(pullerUid, EntityManager)));
```
Comment (MilenVolf) [inline Content.Shared/Movement/Pulling/Systems/PullingSystem.cs:519]:
```suggestion
            Loc.GetString("getting-pulled-popup", ("puller", Identity.Entity(pullerUid, EntityManager)));
```

File: Resources/Locale/en-US/movement/pulling.ftl [hunk 2]
```diff
@@ -0,0 +1 @@
+﻿getting-pulled-popup = { CAPITALIZE(THE($pulling)) } begins pulling you.
```
Comment (MilenVolf) [inline Resources/Locale/en-US/movement/pulling.ftl:1]:
```suggestion
﻿getting-pulled-popup = { CAPITALIZE(THE($puller)) } begins pulling you.
```

**Review by MilenVolf — CHANGES_REQUESTED** [review]
Good PR, just one nitpick

**Review by lzk228 — APPROVED** [review]
nice, as i can tell

**Review by MilenVolf — APPROVED** [review]

**Comment by lzk228** [comment]
> Does this show the persons true name or their visible identity?

it uses identity

**Review by slarticodefast — APPROVED** [review]
Good QOL improvement 👍 
Tested and works as intended.

**Comment by chromiumboy** [comment]
Nice. Test in-game and it works as advertised

**Pull request merged**